### DISCUSSION TITLE
SO 2.4.10 Standalone airgap installer no advance option

### DISCUSSION THREAD

**Discussion by TotieBash** [Security-Onion-Solutions/securityonion#11066] [2.4] [answered]
Hello SO team congrats on 2.4 release...

The past 2.3 installer takes you to advance geek knobs that walks you through like what metadata you want to use zeek or Suri; and how many CPU you want to give for Zeek; etc...  Was this moved to a separate script? it is not on the so_setup.sh anymore...  Please advise..

**Comment by TOoSmOotH** [comment, marked as answer]
Everything is done in the web configuration interface in 2.4. Setup is designed to be simple and streamlined.
